### ISSUE TITLE
Error message in Inventory module of odoo 9.0 when Language is Chinese(CN).

### ISSUE THREAD

**Issue by thisisdong** [odoo/odoo#10444]
```
Odoo Server Error
Traceback (most recent call last):
  File "C:\Odoo\server\.\openerp\http.py", line 643, in _handle_exception
  File "C:\Odoo\server\.\openerp\http.py", line 680, in dispatch
  File "C:\Odoo\server\.\openerp\http.py", line 316, in _call_function
  File "C:\Odoo\server\.\openerp\service\model.py", line 118, in wrapper
  File "C:\Odoo\server\.\openerp\http.py", line 309, in checked_call
  File "C:\Odoo\server\.\openerp\http.py", line 959, in __call__
  File "C:\Odoo\server\.\openerp\http.py", line 509, in response_wrap
  File "C:\Odoo\server\openerp\addons\web\controllers\main.py", line 892, in call_kw
  File "C:\Odoo\server\openerp\addons\web\controllers\main.py", line 884, in _call_kw
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "C:\Odoo\server\.\openerp\api.py", line 354, in old_api
  File "C:\Odoo\server\openerp\addons\web_planner\models\web_planner.py", line 45, in render
  File "C:\Odoo\server\.\openerp\api.py", line 248, in wrapper
  File "C:\Odoo\server\.\openerp\api.py", line 574, in new_api
  File "C:\Odoo\server\openerp\addons\web_editor\models\ir_ui_view.py", line 29, in render
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "C:\Odoo\server\openerp\addons\base\ir\ir_ui_view.py", line 1070, in render
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "C:\Odoo\server\openerp\addons\base\ir\ir_qweb.py", line 252, in render
  File "C:\Odoo\server\openerp\addons\base\ir\ir_qweb.py", line 172, in get_template
  File "C:\Odoo\server\openerp\addons\base\ir\ir_ui_view.py", line 1068, in loader
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "C:\Odoo\server\openerp\addons\base\ir\ir_ui_view.py", line 943, in read_template
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "<string>", line 2, in _read_template
  File "C:\Odoo\server\.\openerp\tools\cache.py", line 85, in lookup
  File "C:\Odoo\server\openerp\addons\base\ir\ir_ui_view.py", line 928, in _read_template
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "C:\Odoo\server\openerp\addons\base\ir\ir_ui_view.py", line 640, in read_combined
  File "lxml.etree.pyx", line 3032, in lxml.etree.fromstring (src\lxml\lxml.etree.c:68106)
  File "parser.pxi", line 1785, in lxml.etree._parseMemoryDocument (src\lxml\lxml.etree.c:102455)
  File "parser.pxi", line 1673, in lxml.etree._parseDoc (src\lxml\lxml.etree.c:101284)
  File "parser.pxi", line 1074, in lxml.etree._BaseParser._parseDoc (src\lxml\lxml.etree.c:96466)
  File "parser.pxi", line 582, in lxml.etree._ParserContext._handleParseResultDoc (src\lxml\lxml.etree.c:91275)
  File "parser.pxi", line 683, in lxml.etree._handleParseResult (src\lxml\lxml.etree.c:92461)
  File "parser.pxi", line 622, in lxml.etree._raiseParseError (src\lxml\lxml.etree.c:91757)
XMLSyntaxError: EntityRef: expecting ';', line 413, column 46
```

**Comment by nocoolyoyo** [comment]
+1

**Comment by elicoidal** [comment]
This is probably a translation issue

**Comment by thisisdong** [comment]
I See. But how can I fix it?

**Comment by elicoidal** [comment]
In short term in your computer you can manually fix the PO file but in long term, you need to identify the xml file and line that is blocking then go to transifex.com change the faulty translation.
Next time transifex is gonna be synchronized the problem will be fixed in the core

**Comment by mart-e** [comment]
Hello,

See my reply [on the forum](https://www.odoo.com/forum/help-1/question/greek-translation-bug-in-sales-menu-95169#answer_95229) for explanations and ways to fix it.
The proper way is to fix it on Transifex.

However, yesterday, I made another fix, rev 6872aae8ef9c1a1fc6a8283aa9d12ce3a6859157, that actually hides the error in case of a wrongly translated planner.
This way, we don't blame the users for the errors of translators ;-)
